docs/migrating-from-legacy-packages.md

```diff
@@ -0,0 +1,606 @@
+
+```js title=.storybook/preview.js
+import 'happo-plugin-storybook/register';
+import happoDecorator from 'happo-plugin-storybook/decorator';
+```
+
+**After (new):**
+
+```js title=.storybook/preview.js
+import 'happo/storybook/register';
+import happoDecorator from 'happo/storybook/decorator';
+```
+
+#### Cypress Integration
+
+The configuration changes for Cypress integration:
+
+**Before (legacy):**
+
+```js title=.happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  targets: {
+    chrome: new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+  },
+
+  // ... rest of config
+};
+```
+
+**After (new):**
+
+```js title=happo.config.ts
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  integration: {
+    type: 'cypress',
+  },
+
+  targets: {
+    chrome: {
+      browserType: 'chrome',
+      viewport: '1024x768',
+    },
+  },
+
+  // ... rest of config
+});
+```
+
+In your Cypress test files, update your imports to use the new package:
+
+**Before (legacy):**
+
+```js title=cypress/support/commands.js
+import 'happo-cypress';
+```
+
+**After (new):**
+
+```js title=cypress/support/commands.js
+import 'happo/cypress';
+```
+
+And in your `cypress.config.js`, update the task import:
+
+**Before (legacy):**
+
+```js title=cypress.config.js
+const happoTask = require('happo-cypress/task');
+```
+
+**After (new):**
+
+```js title=cypress.config.js
+const happoTask = require('happo/cypress/task');
+```
+
+#### Playwright Integration
+
+The configuration changes for Playwright integration:
+
+**Before (legacy):**
+
+```js title=.happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  targets: {
+    chrome: new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+  },
+};
+```
+
+**After (new):**
+
+```js title=happo.config.ts
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  integration: {
+    type: 'playwright',
+  },
+
+  targets: {
+    chrome: {
+      browserType: 'chrome',
+      viewport: '1024x768',
+    },
+  },
+});
+```
+
+In your Playwright test files, update your imports to use the new package:
+
+**Before (legacy):**
+
+```js title=tests/test.spec.js
+import { test } from 'happo-playwright';
+```
+
+**After (new):**
+
+```js title=tests/test.spec.js
+import { test } from 'happo/playwright';
+```
+
+#### Happo Examples Integration
+
+The Happo examples integration has been removed in favor of the static
+integration (originally in the `happo-static` package).
+
+TODO: Provide migration advice for this
+
+### E2E Command Changes
+
+For Cypress and Playwright integrations, use the new `happo` CLI with the `e2e`
+command:
+
+**Before (legacy):**
+
+```bash
+npx happo-e2e -- npx cypress run
+npx happo-e2e -- npx playwright test
+```
+
+**After (new):**
+
+```bash
+npx happo -- cypress run
+npx happo -- playwright test
+```
+
+### Configuration Options
+
+These options are removed:
```

Comment:
Suggestion: "These options, that were only used by Happo Examples integrations, are removed:"